templates/shield-study/src/manifest.json

```diff
@@ -5,7 +5,7 @@
   "manifest_version": 2,
   "applications": {
     "gecko": {
-      "id": "{{fill in your thing}}",
+      "id": "{{id}}",
```

Comment:
This fixed #1; "[FAILED] addMany Missing helper: 'fill'" since **plop** was thinking that `fill` was a helper and not just placeholder text.